src/mmw/js/src/modeling/views.js

```diff
@@ -805,7 +805,10 @@ var ResultsView = Marionette.LayoutView.extend({
                 if (err && err.timeout) {
                   tmvModel.setTimeoutError();
                 } else {
-                  tmvModel.setError(err);
+                    var message = err.error === 'NO_LAND_COVER' ?
```

Comment:
I wonder if we might want to pass along the specific error message for other error/exceptions thrown from Django. I believe many of them currently have error messages set, but I don't know whether these were written to be meaningful for end users.

We could probably address https://github.com/WikiWatershed/model-my-watershed/issues/1187 by sending back a similar error message.